sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryStorageQuerySource.java

```diff
@@ -37,7 +38,7 @@
 @SuppressWarnings({
   "nullness" // TODO(https://issues.apache.org/jira/browse/BEAM-10402)
 })
-public class BigQueryStorageQuerySource<T> extends BigQueryStorageSourceBase<T> {
+class BigQueryStorageQuerySource<T> extends BigQueryStorageSourceBase<T> {
```

Comment:
Why make this package-private?

Reply:
That change was on PR [10369](https://github.com/apache/beam/pull/10369) and I think is about this [discussion](https://github.com/apache/beam/pull/10369#discussion_r364012529), although that specific class is not mentioned in the comments.

Reply:
Got it, thank you